### ISSUE TITLE
IDE0079 should not warn when suppressing IDE0060

### ISSUE THREAD

**Issue by coldacid** [dotnet/roslyn#49669]
**Version Used**: VS 16.8.2, 

**Steps to Reproduce**:

1. Write a method with an unused parameter:
    ```csharp
    [SuppressMessage("Style", "IDE0060:Remove unused parameter")]
    private void SomePrivateMethod(Foo a, Bar b)
    {
        a.Frobnicate();
    }
    ```
2. Run code analysis on solution.

**Expected Behavior**:

SuppressMessage attribute does not cause an IDE0079 message in the Error List and appears normally formatted in the IDE.

**Actual Behavior**:

SuppressMessage attribute causes an IDE0079 message in the Error List, and is displayed as faded out.

**Comment by mavasani** [comment]
I am unable to repro this: 
![IDE0079_FP](https://user-images.githubusercontent.com/10605811/103904930-cd146a00-50b2-11eb-8887-d5946894db34.gif)
